common/protob/messages-management.proto

```diff
@@ -58,6 +58,23 @@ message Features {
     optional bool unfinished_backup = 27;       // report unfinished backup (equals to Storage.unfinished_backup)
     optional bool no_backup = 28;               // report no backup (equals to Storage.no_backup)
     optional bool recovery_mode = 29;           // is recovery mode in progress
+    repeated Feature features = 30;             // list of supported features
+    enum Feature {
+        Feature_Bitcoin = 1;
+        Feature_Altcoins = 2;
+        Feature_Binance = 3;
+        Feature_Cardano = 4;
+        Feature_Crypto = 5;
+        Feature_EOS = 6;
+        Feature_Ethereum = 7;
+        Feature_Lisk = 8;
+        Feature_Monero = 9;
+        Feature_NEM = 10;
+        Feature_Ripple = 11;
+        Feature_Stellar = 12;
+        Feature_Tezos = 13;
+        Feature_U2F = 14;
```

Comment:
Maybe `webauthn` instead? Since that's what we call the app

Reply:
I want to make U2F and WebAuthn as two different features. Currently only U2F is supported. WebAuthn is in a branch yet to be merged (and we'll add Feature_WebAuthn once this is merged). Also a good indication why Features do not equal Apps (as one app can contain more features).

Reply:
Makes sense.